deploy/scripts/build_and_push

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+
+require 'json'
+require 'base64'
+
+secrets_json = `kubectl get secrets -n hmcts-complaints-formbuilder-adapter-production ecr-repo-hmcts-complaints-adapter -o json`
+
+hash = JSON.parse(secrets_json)
+
+access_key = Base64.strict_decode64(hash['data']['access_key_id'])
+secret_access_key = Base64.strict_decode64(hash['data']['secret_access_key'])
+repo_url = Base64.strict_decode64(hash['data']['repo_url'])
+
+puts 'Building docker image...'
+out = `docker build -t #{repo_url}:latest .`
+puts out
+
+puts 'Logging into AWS ECR...'
+command = "AWS_ACCESS_KEY_ID=#{access_key} AWS_SECRET_ACCESS_KEY=#{secret_access_key} aws ecr get-login --no-include-email --region eu-west-2"
+out = `#{command}`
+`#{out}`
+
+puts 'Pushing docker image...'
+out = `AWS_ACCESS_KEY_ID=#{access_key} AWS_SECRET_ACCESS_KEY=#{secret_access_key} docker push #{repo_url}:latest`
```

Comment:
you don't want this running on circleci. the output that is

Reply:
Cool yeah, can confirm that running this locally does not dump anything sensitive to the logs

Reply:
👍